common/git_worker.py

```diff
@@ -236,18 +239,27 @@ def extract_all_repos(self):
                 repo.revert_commit_by_time(commit_timestamp)
                 repo.checkout()
 
-    def save_repo_states(self, sources_file):
+    def save_repo_states(self, sources_file, trigger):
         """
         Write repositories states to json file
 
         :param sources_file: path to json file
+        :type sources_file: pathlib.Path
+
+        :param trigger: Triggered repository
+        :type trigger: String
         """
 
         with sources_file.open('a') as sources_state:
             states = {}
             for state in self.repo_states:
                 states[state.name] = {
                     'branch': state.branch,
-                    'commit_id': state.commit_id
+                    'commit_id': state.commit_id,
+                    'url': state.url
                 }
-            sources_state.write(json.dumps(states))
+
+                if trigger == state.name:
```

Comment:
it can be added for all repos as True or False
'trigger': True if trigger == state.name else False